src/component/FooterComponent/Footer.jsx

```diff
@@ -0,0 +1,27 @@
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import * as React from 'react';
+import Copyright from './Copyright/Copyright';
+import useStyles from './footerStyle';
+import Home from './Home/Home';
+import Info from './Info/Info';
+import Logo from './Logo/logo';
+import SocialMedia from './SocialmediaIcons/SocialmediaIcons';
+
+export default function ButtonAppBar() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <AppBar position="static" color="inherit">
```

Comment:
isn't the position="static" the one that makes it sticky to the screen? we don't really want that, only the top navbar